pkgs/development/python-modules/fastai/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, fastprogress
+, fastcore
+, fastdownload
+, torchvision-bin
```

Comment:
Why `torchvision-bin` instead of `torchvision`?

Reply:
torchvision-bin supports cuda. when I did with torch. 'torch.cuda.is_available()` evaluated to false

Reply:
Hmm, torchvision-bin doesn't seem correct.

The default `torch` package doesn't depend on CUDA (see `torchWithCuda`).  However, for license reasons this means `torchWithCuda` and any packages depending on it won't be built on Hydra.  Probably it's best to either (a) depend on `torch` and then users can override to set `torch = torch{With/Without}Cuda` or (b) depend on a `torch` only in `buildInputs` and let users choose which torch is present at runtime (as with `torchmetrics`) but I don't know which is preferable.

Reply:
ok. I will change to torchvision then.